.phan/config.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * This configuration will be read and overlaid on top of the
+ * default configuration. Command line arguments will be applied
+ * after this file is read.
+ */
+
+return [
```

Comment:
should we put this in a common location like the coding standards repo ?

Reply:
I would not suggest this - as these settings are depending on the layout of the app (where will code be). And the settings are definitely not the same for every app. Some apps we can have very strict settings fast, whereas some apps will require more lenient settings by design